extensions/wikia/PowerUser/PowerUser.class.php

```diff
@@ -174,7 +179,7 @@ public function removePowerUserRemoveGroup( $sProperty ) {
 			&& $this->bUseGroups
 			&& $this->isGroupForRemoval( $sProperty )
 		) {
-			\UserRights::removeGlobalGroup( $this->oUser, self::GROUP_NAME );
+			$this->userPermissions()->removeUserFromGroup( $this->oUser, self::GROUP_NAME );
```

Comment:
... and here `userPermissions()` - is something different returned or it's just in the middle of rename?